src/npm.ts

```diff
@@ -0,0 +1,29 @@
+import { Job } from "@brigadecore/brigadier";
+
+export const npmReleaseImg = "brigadecore/npm-release:edge";
```

Comment:
Are we ok with always running the `:edge` tag?

Reply:
Good point.  I've created a follow-up around this area: https://github.com/brigadecore/brigade-utils/issues/47